src/main/kotlin/gg/skytils/skytilsmod/features/impl/handlers/Waypoints.kt

```diff
@@ -370,3 +383,32 @@ data class Waypoint @OptIn(ExperimentalSerializationApi::class) constructor(
         UGraphics.enableDepth()
     }
 }
+
+@Serializable
+data class Coleweight(
```

Comment:
Maybe treat these like generic waypoints? have default values for r, g, b so that with just x, y, z it'll be accepted

Reply:
should the options (basically only name) also have a default value?